env0/resource_environment_import.go

```diff
@@ -58,6 +58,12 @@ func resourceEnvironmentImport() *schema.Resource {
 				Description: "iac version of the environment",
 				Optional:    true,
 			},
+			"soft_delete": {
+				Type:        schema.TypeBool,
+				Description: "soft delete the configuration variable, once removed from the configuration it won't be deleted from env0",
```

Comment:
Is this comment correct? default is false...

Reply:
yeah if they set the field ( to true ) it won't delete
I just defaulted it to false so it won't be set by default ( we want soft delete to be explicit )

we generate the code that uses these flags so we'll just set `soft_delete = true` we don't really expect customers to use those flags, but we just didn't want the resource to act in unexpected ways without it being explicit